bigtable/api/Snippets/Reads.cs

```diff
@@ -137,7 +144,10 @@ public string readRowRanges(string projectId = "YOUR-PROJECT-ID", string instanc
             ReadRowsStream readRowsStream = bigtableClient.ReadRows(tableName, rowSet);
```

Comment:
We should probably have using statements for all of these streams...

Reply:
Oh didn't see this before merging, but yep, I'll do that.

Reply:
ReadRowStream is not disposable, I've created b/352471768.